### ISSUE TITLE
open telemetry instrumentation

### ISSUE THREAD

**Issue by adamfoneil** [adamfoneil/OpenTelemetryDemo#2]
Couple things I'd like to achieve here:
- make it so EF traffic is captured in Jaeger with SQL statements and parameters
- make it so http traffic is captured as well with request and responses.

Please use [apicalls.http](https://github.com/adamfoneil/OpenTelemetryDemo/blob/master/OpenTelemetryDemo/apicalls.http) for methods to try. I'm wanting to be able to see traces in Jaeger that include the http request info with the EF traffic nested within it.

You should be able to create local SQL Server db via migration.
The Jaeger image I'm using is `jaegertracing/all-in-one:latest` a0253dd1a194

**Comment by wasabii** [comment]
Got it all working, except parameters in EF, since they aren't supported by the EF Core OTEL implementation. You'd have to add them manually. It's only coded to log the command text.

**Comment by adamfoneil** [comment]
thank you @wasabii did you push changes? I'm not seeing any. What would manual param OTEL logging look like?

**Comment by wasabii** [comment]
Haven't pushed yet. Found an issue where the ASP.NET request logging just stops working after a minute. Trying to figure that out.

I think I might have been confused on exactly what you were asking for on a few of these things. Because I stumbled across the middleware that was built which is trying to load the ENTIRE request body into tracing? That's not a good idea generally. Requests are of unknown size, and not necessarily text data, etc.

Was the goal to log the entire request BODY? Or just select headers, etc?
